test/database/create.mjs

```diff
@@ -0,0 +1,105 @@
+//@ts-check
+
+import * as awilix from "awilix";
+import { expect } from "chai";
+import chalk from "chalk";
+import { fql, ServiceError } from "fauna";
+import sinon from "sinon";
+
+import { builtYargs, run } from "../../src/cli.mjs";
+import { setupTestContainer as setupContainer } from "../../src/config/setup-test-container.mjs";
+
+describe("database create", () => {
+  let container, logger, runV10Query;
+
+  beforeEach(() => {
+    // reset the container before each test
+    container = setupContainer();
+    logger = container.resolve("logger");
+    runV10Query = container.resolve("runV10Query");
+  });
+
+  [
+    { missing: "name", command: "database create --secret 'secret'" },
+    { missing: "secret", command: "database create --name 'name'" },
+  ].forEach(({ missing, command }) => {
+    it(`requires a ${missing}`, async () => {
+      try {
+        await run(command, container);
+      } catch (e) {}
+
+      const message = `${chalk.reset(await builtYargs.getHelp())}\n\n${chalk.red(
+        `Missing required argument: ${missing}`,
+      )}`;
+      expect(logger.stderr).to.have.been.calledWith(message);
+      expect(container.resolve("parseYargs")).to.have.been.calledOnce;
+    });
+  });
+
+  [
+    {
+      args: "--name 'testdb' --secret 'secret'",
+      expected: { name: "testdb", secret: "secret" },
+    },
+    {
+      args: "--name 'testdb' --secret 'secret' --typechecked",
+      expected: { name: "testdb", secret: "secret", typechecked: true },
+    },
+    {
+      args: "--name 'testdb' --secret 'secret' --protected",
+      expected: { name: "testdb", secret: "secret", protected: true },
+    },
+    {
+      args: "--name 'testdb' --secret 'secret' --priority 10",
+      expected: { name: "testdb", secret: "secret", priority: 10 },
+    },
+  ].forEach(({ args, expected }) => {
+    it(`calls runV10Query with correct arguments: ${args}`, async () => {
```

Comment:
thoughts on structuring this to be describe the test in a more user-facing & sentence-like way?

thinking something like:
```
describe(`calls fauna with the user-specified arguments`, () => {
  ...
  it(`including ${Object.keys(expected).join(', ')}.`, async () => {
  ...
  })
})
```

Reply:
yup can do